app/helpers/js_helper.rb

```diff
@@ -42,10 +42,6 @@ def javascript_hide_if_exists(element)
     "if (miqDomElementExists('#{j_str(element)}')) #{javascript_hide(element)}".html_safe
   end
 
-  def partial_replace(from, partial, locals)
-    "$(\"##{h(from)}\").replaceWith(\"#{escape_javascript(render(:partial => partial, :locals => locals))}\");".html_safe
```

Comment:
I have no idea how this ever worked because the caller never called with the third paramter.